changes/178.feature

```diff
@@ -0,0 +1 @@
+Add 'agent-list' option. User can manually-assigned agent list to session creation API
```

Comment:
```suggestion
Add `--agent-list` option to `run` and `start` commands so that superadmins can manually assign the agent(s) for new sessions to debug and diagnose the cluster and scheduler.
```
Give concrete context in a single sentence. And this is where to use backticks!